backend/src/main/java/develup/api/DiscussionCommentApi.java

```diff
@@ -29,9 +29,10 @@ public DiscussionCommentApi(DiscussionCommentService discussionCommentService) {
 
     @GetMapping("/discussions/{discussionId}/comments")
     @Operation(summary = "디스커션 댓글 목록 조회 API", description = "디스커션 댓글 목록을 조회합니다.")
-    public ResponseEntity<ApiResponse<List<GroupingDiscussionCommentResponse>>> getGroupedComments(@PathVariable Long discussionId) {
-        var responses = discussionCommentService.getGroupingComments(discussionId);
-        return ResponseEntity.ok(new ApiResponse<>(responses));
+    public ResponseEntity<ApiResponse<List<GroupingDiscussionCommentResponse>>> getGroupedComments(
+            @PathVariable Long discussionId
+    ) {
+        return ResponseEntity.ok(new ApiResponse<>(discussionCommentService.getGroupingComments(discussionId)));
```

Comment:
[Request Changes]
```suggestion
List<GroupingDiscussionCommentResponse>> responses = discussionCommentService.getGroupingComments(discussionId);

return ResponseEntity.ok(new ApiResponse<>(responses));
```

한 줄이라도 변수로 분리하는 컨벤션입니다.
가급적 이런 컨벤션은 맞춰주시는 게 좋을 것 같습니다. 😊